sdk/servicebus/azure-servicebus/azure/servicebus/management/_models.py

```diff
@@ -1020,3 +1088,69 @@ def _to_internal_entity(self):
 }  # type: Dict[Type[Model], Type]
 EMPTY_RULE_ACTION = InternalEmptyRuleAction()
 TRUE_FILTER = TrueRuleFilter()
+
+
+class AuthorizationRule(object):
+    """Authorization rule of an entity.
+
+    :param type: The authorization type.
+    :type type: str
+    :param claim_type: The claim type.
+    :type claim_type: str
+    :param claim_value: The claim value.
+    :type claim_value: str
+    :param rights: Access rights of the entity. Values are 'Send', 'Listen', or 'Manage'.
+    :type rights: list[AccessRights]
+    :param created_at_utc: The date and time when the authorization rule was created.
+    :type created_at_utc: ~datetime.datetime
+    :param modified_at_utc: The date and time when the authorization rule was modified.
+    :type modified_at_utc: ~datetime.datetime
+    :param key_name: The authorization rule key name.
+    :type key_name: str
+    :param primary_key: The primary key of the authorization rule.
+    :type primary_key: str
+    :param secondary_key: The primary key of the authorization rule.
+    :type secondary_key: str
+    """
```

Comment:
apart from `param`, should we also add `ivar` in docstring?
this comment applies to each model class.

Reply:
Did a brief eyeballing of our docs, and yes, it does seem like we ought to.  I've added this as a line item to [this issue](https://github.com/Azure/azure-sdk-for-python/issues/11684) since we can do that next sprint without issue. (and already had the pending docstring work.)